src/Component.svelte

```diff
@@ -160,7 +160,7 @@
       {max}
       {placeholder}
       defaultDate={defaultValue}
-      {disabled}
```

Comment:
I think this needs to be an or, i.e.

`disabled={fieldState?.disabled || disabled}`